contrib/gitian/gitian-build.py

```diff
@@ -14,14 +17,21 @@ def setup():
         programs += ['lxc', 'debootstrap']
     if not args.no_apt:
         subprocess.check_call(['sudo', 'apt-get', 'install', '-qq'] + programs)
-    if not os.path.isdir('gitian.sigs'):
-        subprocess.check_call(['git', 'clone', 'https://github.com/monero-project/gitian.sigs.git'])
-    if not os.path.isdir('gitian-builder'):
-        subprocess.check_call(['git', 'clone', 'https://github.com/devrandom/gitian-builder.git'])
-    if not os.path.isdir('monero'):
-        subprocess.check_call(['git', 'clone', 'https://github.com/monero-project/monero.git'])
-    os.chdir('gitian-builder')
+    if not os.path.isdir('sigs'):
+        subprocess.check_call(['git', 'clone', gsigs, 'sigs'])
+    if not os.path.isdir('builder'):
+        subprocess.check_call(['git', 'clone', gbrepo, 'builder'])
+    os.chdir('builder')
+    subprocess.check_call(['git', 'config', 'user.email', 'gitianuser@localhost'])
+    subprocess.check_call(['git', 'config', 'user.name', 'gitianuser'])
     subprocess.check_call(['git', 'checkout', '963322de8420c50502c4cc33d4d7c0d84437b576'])
+    subprocess.check_call(['git', 'fetch', 'origin', '72c51f0bd2adec4eedab4dbd06c9229b9c4eb0e3'])
```

Comment:
Just to note, this commit has been merged upstream https://github.com/devrandom/gitian-builder/pull/219

Reply:
ping @TheCharlatan

Reply:
The reason why we pin a certain commit hash is to ensure that we all use the same gitian scripts for building. I think we should update to latest master (or at least to this merge).